doc/content/integrations/cloud-integrations/thinger-io/_index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Thinger.io"
+description: ""
+weight: 
+---
+
+[Thinger.io](https://thinger.io/) is a cloud IoT Platform that provides every needed tool to prototype, scale and manage connected products in a very simple way. Our goal is to democratize the use of IoT making it accessible to the whole world, and streamlining the development of big IoT projects.
+
+From [Thinger.io](https://thinger.io/) we wanted to offer an improved integration to {{% tts %}} users by providing easy to configure tools for the storing, analyzing and showing devices data in a simple way. [The plugin](https://docs.thinger.io/plugins/the-things-stack) allows to retrieve {{% tts %}} webhook messages to enhance the integration with some interesting features such as:
```

Comment:
All fine, but can you please rephrase the **we** and **our** in these sentences? It's a bit weird to use **we** for Thinger.io when it's a TTS/TTI webpage